src/backends/perf_pt/decode.c

```diff
@@ -170,27 +170,27 @@ perf_pt_init_block_decoder(void *buf, uint64_t len, int vdso_fd, char *vdso_file
 }
 
 /*
- * Updates `*addr` and `*len` with the start address and the length of the next
- * block in the instruction stream.
+ * Updates `*first_instr` and `*last_instr` with the address of the first and last
+ * instructions of the next block in the instruction stream.
  *
- * A start address of 0 indicates that the end of the instruction stream has
- * been reached.
+ * An instruction address of 0 indicates that the end of the instruction stream
```

Comment:
Which address is set to zero? The `first_instr`?

Reply:
The first, but actually I think we should set both to zero.

Reply:
That's fine, but let's document it.